ArduPlane/mode.cpp

```diff
@@ -201,7 +201,22 @@ void Mode::update_target_altitude()
     } else if (plane.target_altitude.offset_cm != 0 && 
                !plane.current_loc.past_interval_finish_line(plane.prev_WP_loc, plane.next_WP_loc)) {
         // control climb/descent rate
+#if HAL_QUADPLANE_ENABLED
+        // On climbout we will give the ability to do a full rate climb after the VTOL Takeoff. We
+        // will set the glide slope proportion to the destination to achieve this.
+        if (quadplane.get_do_full_rate_climbout_enable() && quadplane.get_do_full_rate_climbout()) {
```

Comment:
Why is this quadplane-specific?  Planes do have takeoffs which ytou might want a full-rate climbout?  Those trees can hurt regardless of hte number of props you have.